modular_skyrat/modules/decay_subsystem/code/decaySS.dm

```diff
@@ -104,9 +104,9 @@ SUBSYSTEM_DEF(decay)
 				if(!iterating_floor.Enter(spawned_web))
 					qdel(spawned_web)
 
-			if(prob(NEST_PERCENT_CHANCE * severity_modifier) && prob(50))
+			if(prob(NEST_PERCENT_CHANCE * severity_modifier) && prob(50) && !CONFIG_GET(flag/ssdecay_disable_nests))
```

Comment:
That'll probably be much faster than doing like two probs and then checking the config.
```suggestion
			if(!CONFIG_GET(flag/ssdecay_disable_nests) && prob(NEST_PERCENT_CHANCE * severity_modifier) && prob(50))
```